src/app/static/src/tests/components/baseline/baseline.test.ts

```diff
@@ -194,6 +194,34 @@ describe("Baseline upload component", () => {
         expectDeleteToDispatchAction(2, () => actions.deletePopulation, done);
     });
 
+    it("upload can set validAdr to true if data is from ADR", async () => {       
+        const store = createSut({country: "Malawi"});
+        const wrapper = shallowMount(Baseline, {store});
+
+        wrapper.findAll(ManageFile).at(0).setProps({hasValidAdr : true});
+        expect(wrapper.findAll(ManageFile).at(0).props().hasValidAdr).toBe(true);
```

Comment:
Rather than testing whether props are as you expect, it's always better to test that the UI element itself is rendered correctly - in this case, we should test whether the tag with "ADR" in it is rendered. In fact, the code has a bug in it and the expected element is not rendered, but you can't pick that up from this test.

Reply:
We should also have similar tests for the `SurveyAndProgram.vue` component